Selenium.Axe/Selenium.Axe/IncludeExcludeManager.cs

```diff
@@ -1,97 +0,0 @@
-﻿using Newtonsoft.Json;
-using Newtonsoft.Json.Serialization;
-using System;
-using System.Collections.Generic;
-using System.Linq;
-
-namespace Selenium.Axe
-{
-    /// <summary>
-    /// Handle all initialization, serialization and validations for includeExclude aXe object.
-    /// For more info check this: https://github.com/dequelabs/axe-core/blob/master/doc/API.md#include-exclude-object
-    /// </summary>
-    public class IncludeExcludeManager
```

Comment:
Removing a public class is, technically, a breaking change. Maybe we should leave the `IncludeExcludeManager` removal for a separate PR?

(we're also losing-without-replacement the `ValidateParameters` functionality)

Reply:
IncludeExcludeManager is being used to create axe run context. 
Even though IncludeExcludeManager is public, AxeBuilder doesnt expose any api to create the context object. User need to use the include / exclude apis

Reply:
but, i can add the validateParameters to the AxeBuilder